frontend/src/models/models.js

```diff
@@ -67,23 +68,31 @@ module.exports = app => app.component('models', {
       this.sortDocs(num, path);
     }
 
+
     if (this.currentModel != null) {
       await this.getDocuments();
```

Comment:
So the only issue right now is that because mounted is calling `getDocuments()` the fields get overwritten which is why the helper has to be called again. Let me know if you want a different approach.